package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amplify-category-video",
-  "version": "2.7.3",
```

Comment:
I'm still confused on our version bumps, we should probably get into the habit of summarizing any version updates in the "releases" notes.